### ISSUE TITLE
[QUESTION] What's the status of this project?

### ISSUE THREAD

**Issue by jkudish** [laraform/laraform#43]
Hey there @laraform!

First of all, thank you for producing such a time saving and awesome package. I'm excited by the prospect of Laraform. Having played around with the community edition, it feels like a solid solution and I'm excited by the potential of it. However, looking at the slower github activity in the last few months, and seeing as how the project seems to be primarily run by a single person, I am curious to know what the project's status is looking like.

We are about to embark on a fairly serious rewrite of some of our forms for one of our clients, and I'd love to use Laraform to do this. However, I do worry about the longer term maintainability (or lack thereof) with this project. I'm trying to keep an open mind and fully know and understand how sometimes these things just don't work out. I mean no disrespect and no hate, just wanting to know where this project is in terms of production readyness and commercial viablity before we put all our eggs into this basket.

Thanks a bunch!

**Comment by laraform** [comment]
Hi @jkudish!

Thank you for bringing up this topic!

First of all, I'm flattered that you are considering Laraform for such change with your client. I mean that's the goal of the whole project but having these feedback from you and other guys already is really encouraging.

Yes, we're in beta because I felt the first step is to put out the product and see how people react. That was in May 2019 and we had our first campaign at the end of November, which has brought quite some attention. Since that I'm convinced that it worth shifting my focus towards and I've already started reorganizing my stuff to give it 100%.

It looks like I've been missing for the past one and a half month, but besides having some rest at the end of the year, I'm closing up my previous projects and planning for 2020. I can see a long road ahead Laraform but given the fact that it can affect the work of a lot of developers, I approach this carefully. The "beta" sign is there to show everyone to use it with caution because this is an early product. It has already been used in production for some custom software projects (including ones I've worked on) and I can definitely see the areas which could be improved. I wouldn't say it's buggy or not ready for production but some parts could just be better built for ease of use. I will be comfortable removing the "beta" sign once all the tests are in place and improvements I have in mind are made.

2020 will be about becoming stable. Besides adding some necessary features (like Tailwind CSS, code splitting, etc.) and working on educational materials, I will focus on writing tests, improving codebase, handling issues and listening to feedback. I'm also planning to open up the space for fellow developers to contribute and see what happens.

Well, at the moment I'm working on this alone, apart from having some minor help here and there. By the end of the month I will be into it with 100% focus again, but there's a lot of things to do. I'm not in a rush to expand the team but if I happen to find someone with strong work ethics and high coding standards that might be a good match and a way to speed things up.

As for now, I'd recommend playing around with the library yourself to see what its capabilities are. If you like it at its current state, then I'd say go ahead, but expect no more for a year than what it can do right now besides some milder improvements and more stable grounds.

Hope this helps!
